icicle/CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ if (NOT IS_CURVE_SUPPORTED)
   message( FATAL_ERROR "The value of CURVE variable: ${CURVE} is not one of the supported curves: ${SUPPORTED_CURVES}" )
 endif ()
 
+if (DEVMODE STREQUAL "ON")
```

Comment:
not big deal but note that it is recommended to use cmake-option compared to strings since it's a boolean value. It's cleaner since it can take only two values and not any string but also because you declare options on top and it's easier to understand what options are there.

I can see other values like G2_DEFINED are the same but maybe worth making them all boolean options, unless the rust build.rs scripts cannot use that, which I guess is not the case but not sure about that.

Reply:
@ChickenLover TODO in v2